docs/content/contribution-guidelines/operations/docs-styleguide.md

```diff
@@ -32,9 +32,29 @@ Much of KubeStellar's existing documentation -- especially introductory and over
 The **Design Guide** will include guidance on use and placement of iconography in KubeStellar docs and UIs.
 In textual documentation, however, emojis may interfere with rendering and navigation of the website documentation, _especially_ if they are used in headings or titles. They also may make the documentation inaccessible to visitors who must use screen reader technology.
 
-### Always include alt-text for images
+### Compose and Format for Accessibility
 
-This is a basic requirement for accessibility. Images and diagrams should always have an appropriate alt-text attribute. In general, keep the [W3C Guidelines for Accessibility](https://www.w3.org/TR/WCAG21/) in mind when writing/illustrating KubeStellar documentation 
+Writing for accessibility means ensuring that screen readers can easily read your text, your content is easy to navigate, visually clear and logically structured. This includes the appropriate usage of Heading tags, styling your text consistently and using a 'gender-neutral' language where applicable.
+
+Accessibility is a part of any good web project, and docs are no different. As a bare minimum, we require that all our images and diagrams always have an appropriate 'alt-text' to support visually impared users and screenreaders. In Markdown, you can add alt-text using the following syntax:
+
+```{.bash}

```

Comment:
Why are you writing `{.bash}` as the language tag here, rather than simply `bash`?

The language involved is Markdown, not bash.